src/diet.ts

```diff
@@ -171,7 +174,10 @@ function fillStomach() {
   while (myFullness() + 5 <= fullnessLimit()) {
     if (myMaxhp() < 1000) maximize("0.05hp, hot res", false);
     const count = Math.floor(Math.min((fullnessLimit() - myFullness()) / 5, mySpleenUse() / 5));
-    eatSpleen(count, $item`Ol' Scratch's salad fork`);
+    if (mallPrice(saladFork) < (55 * MPA) / 6) {
```

Comment:
Consider trying to buy at equilibrium price instead of using mallPrice--this way, we can get any 3-per-day limiteds if relevant